dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/WithSpanInterceptor.java

```diff
@@ -55,14 +58,22 @@ public WithSpanInterceptor() {
 
     @Inject
     public WithSpanInterceptor(final OpenTelemetry openTelemetry) {
-        InstrumenterBuilder<MethodRequest, Void> builder = Instrumenter.builder(openTelemetry, instrumentationName, new MethodRequestSpanNameExtractor());
-        MethodSpanAttributesExtractor<MethodRequest, Void> attributesExtractor = MethodSpanAttributesExtractor.newInstance(
+        Instrumenter<MethodRequest, Void> instrumenter = null;
+        try {
+            InstrumenterBuilder<MethodRequest, Void> builder = Instrumenter.builder(openTelemetry, instrumentationName, new MethodRequestSpanNameExtractor());
+            MethodSpanAttributesExtractor<MethodRequest, Void> attributesExtractor = MethodSpanAttributesExtractor.newInstance(
                                                                                                                            MethodRequest::getMethod,
                                                                                                                            new WithSpanParameterAttributeNamesExtractor(),
                                                                                                                            MethodRequest::getArgs);
 
-        this.instrumenter = builder.addAttributesExtractor(attributesExtractor)
+            instrumenter = builder.addAttributesExtractor(attributesExtractor)
                         .buildInstrumenter(methodRequest -> spanKindFromMethod(methodRequest.getMethod()));
+        } catch (Exception e) {
+            Tr.error(tc, Tr.formatMessage(tc, "CEMOT5101.telemetry.error", e));
+            instrumenter = null;
+        } finally {
+            this.instrumenter = instrumenter;
+        }
```

Comment:
I don't think there's a need for a `finally` block here (or for the local variable `instrumenter`)